introduction.md

```diff
@@ -33,48 +34,50 @@ The Task Force found that the majority of success criteria from WCAG 2 can apply
-*   Replacement text that is presented to show how an SC would read as modified by the advice in this document are in `<ins>` elements visually styled as bold green text with a dotted underline.
-*   Notes are slightly inset and begin with the phrase “NOTE”. Each note is in its own inset box styled in pale green with a darker green line on the left side of the box.
-*   References to glossary items from WCAG 2 are presented in `<cite>` elements visually styled as ordinary text with a dotted underline, and contain title attributes noting these are WCAG definitions. They turn blue with a yellow background when mouse or keyboard focus is placed over them.
-*   References to glossary items in this document are presented in `<cite>` elements visually styled as ordinary text with a dark gray underline.
-*   Hereafter, the short title “WCAG2ICT” is used to reference this document.
-*   In headings the term "Success Criterion" has been shortened to “SC” for brevity.
+* Quotes from WCAG 2 and WCAG 2 Understanding Documents are in `<blockquote>` elements and visually styled with a gray bar on the left, and immediately follow the heading for the principle, guideline, or success criterion.
+* Additional guidance provided by this document begins with the phrase “Applying” and has no special visual styling.
+* Replacement text that is presented to show how an SC would read as modified by the advice in this document are in `<ins>` elements visually styled as bold green text with a dotted underline.
+* Notes are slightly inset and begin with the phrase “NOTE”. Each note is in its own inset box styled in pale green with a darker green line on the left side of the box.
+* References to glossary items from WCAG 2 are presented in `<cite>` elements visually styled as ordinary text with a dotted underline, and contain title attributes noting these are WCAG definitions.  They turn blue with a yellow background when mouse or keyboard focus is placed over them.
+* References to glossary items in this document are presented in `<cite>` elements visually styled as ordinary text with a dark gray underline.
+* Hereafter, the short title “WCAG2ICT” is used to reference this document.
+* In headings the term "Success Criterion" has been shortened to “SC” for brevity.
 
 ### Comparison with the 2013 WCAG2ICT Note
 
-<p class="ednote">The WCAG2ICT Task Force has incorporated all of the new WCAG 2.1 guidelines, criteria and glossary terms. The next draft version will incorporate new WCAG 2.2 criteria and glossary terms as well as address open issues on any of the content in the document.</p>
+<div class="ednote"><p>The WCAG2ICT Task Force has incorporated all of the new WCAG 2.1 guidelines, criteria and glossary terms. The next draft version will incorporate new WCAG 2.2 criteria and glossary terms as well as address open issues on any of the content in the document.</p>
```

Comment:
Suggest this editor's note is removed altogether. Not needed now that all of the content is included.